deisctl/backend/fleet/list_units.go

```diff
@@ -82,8 +104,11 @@ func (c *FleetClient) ListUnits() (err error) {
 	}
 
 	for _, us := range unitStates {
-		if strings.HasPrefix(us.Name, "deis-") {
-			states = append(states, us)
+		for _, prefix := range unitPrefixes {
```

Comment:
I thought about making the data structure a Go `map` with empty values just so I wouldn't have to add another loop, but that sounded wrong. I'm assuming this code is more or less optimal after the compiler is done with it. :)